docs/source/item-validation.rst

```diff
@@ -87,6 +87,34 @@ an example of a schema for the quotes item from the :doc:`tutorial </getting-sta
     ]
   }
 
+With Cerberus
+-------------
+
+`Cerberus`_ is a powerful yet simple and lightweight data validation
+tool, designed to be ​extensible​, allowing for custom validation​ and has ​no
+dependencies. You can define what the field contains, what is required, the type of
+each field, as well as dependencies and regex.
+
+.. warning::
+
+   You need to install `cerberus`_ to use this feature.
+
+This `usage`_ and `validation-rules`_ guide explain the main keywords and how to make a
+schema. Here we have an example of a schema for the quotes item from the
+:doc:`tutorial </getting-started>`.
+
+.. code-block:: json
+
+{
```

Comment:
This block is not appearing correctly when we build the docs. Apparently fixing the indentation solve the problem.

Reply:
Roger that. Fixed. Building again. Here's a screenshot.
![Screenshot_2019-08-15_11-09-17](https://user-images.githubusercontent.com/22801822/63074925-4440df80-bf4d-11e9-92ee-87edecce54c0.png)